bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/internal/models/AbstractContainerImpl.java

```diff
@@ -64,4 +83,33 @@
         }
         return items;
     }
+
+    @Nonnull
+    @Override
+    public String getExportedType() {
+        return request.getResource().getResourceType();
+    }
+
+    @Nonnull
+    @Override
+    public Map<String, ? extends ComponentExporter> getExportedItems() {
+        if (childrenModels == null) {
```

Comment:
and `childModels` here

Reply:
I've moved this change to issue/298 branch and it will be merged from a new PR